tests/e2e/cypress/integration/creator-name-and-version-page.spec.ts

```diff
@@ -0,0 +1,11 @@
+/// <reference types="cypress" />
+
+context('Check if the header is displayed', () => {
+  it('the header should display correctly', () => {
+    cy.visit(
+      '/wiki/spaces/KONVIW/pages/238354433/Random+Confluence+Page',
```

Comment:
I can't find ,,random confluence page" and to be honest the name is wired. It's open source project, can You create something like ,,Author and Version" ? and create page in konviw space instead of personal.